extensions/lifecycle/conductor/service.js

```diff
@@ -1,44 +1,82 @@
 'use strict'; // eslint-disable-line
 
+const async = require('async');
 const werelogs = require('werelogs');
+const { errors } = require('arsenal');
+const { DEFAULT_LIVE_ROUTE, DEFAULT_READY_ROUTE } = require('arsenal').network.probe.ProbeServer;
+const { ZenkoMetrics } = require('arsenal').metrics;
 
 const LifecycleConductor = require('./LifecycleConductor');
-const { HealthProbeServer } = require('arsenal').network.probe;
-
 const config = require('../../../lib/Config');
+const { sendSuccess, sendError, startProbeServer } = require('../../../lib/util/probe');
+
 const zkConfig = config.zookeeper;
 const kafkaConfig = config.kafka;
 const lcConfig = config.extensions.lifecycle;
 const repConfig = config.extensions.replication;
 
-const lcConductor = new LifecycleConductor(
-    zkConfig, kafkaConfig, lcConfig, repConfig);
 
-werelogs.configure({ level: config.log.logLevel,
-                     dump: config.log.dumpLevel });
-const logger = new werelogs.Logger('Backbeat:Lifecycle:Conductor:service');
-const healthServer = new HealthProbeServer({
-    bindAddress: config.healthcheckServer.bindAddress,
-    port: config.healthcheckServer.port,
+const lcConductor = new LifecycleConductor(zkConfig, kafkaConfig, lcConfig, repConfig);
+
+werelogs.configure({
+    level: config.log.logLevel,
+    dump: config.log.dumpLevel,
 });
+const logger = new werelogs.Logger('Backbeat:Lifecycle:Conductor:service');
+
+/**
+ * Handle ProbeServer liveness check
+ *
+ * @param {http.HTTPServerResponse} res - HTTP Response to respond with
+ * @param {Logger} log - Logger
+ * @returns {string} response
+ */
+function handleLiveness(res, log) {
+    if (lcConductor.isReady()) {
+        sendSuccess(res, log);
+    } else {
+        log.error('lifecycle conductor is not ready!');
```

Comment:
sendError will do a `log.debug` and because it should return a >300 response it might be enough. Not sure we want to overly log here is all. Open to debate for sure though...

Reply:
Ah, it was a copy-paste from the old implementation.